src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationServiceTest.kt

```diff
@@ -208,7 +208,9 @@ internal class RecommendationServiceTest : ServiceTestBase() {
 
   @Test
   fun `get a recommendation from the database`() {
-    val recommendation = Optional.of(RecommendationEntity(data = RecommendationModel(crn = crn)))
+    val recommendation = Optional.of(
+      RecommendationEntity(data = RecommendationModel(crn = crn, status = Status.DRAFT, recommendation = Recommendation.NO_RECALL, lastModifiedBy = "Bill", lastModifiedDate = "2022-05-18T19:33:56"))
```

Comment:
Should the date be UTC? `2022-05-18T19:33:56.000Z`
https://dsdmoj.atlassian.net/wiki/spaces/MRD/pages/4009590838/3.+How+to+store+times+and+adjusting+for+DST

Reply:
The real Entity from the db will be UTC. The mock here doesn't care, any String will do.